scripts/orka-init.sh

```diff
@@ -23,17 +23,26 @@ chmod 400 /etc/orka-token
 
 # Create VM config
 VM_NAME="tekton-$(openssl rand -hex 4)"
-curl $CURL_FLAGS --request POST "${ORKA_API}/resources/vm/create" \
-  --header 'Content-Type: application/json' \
-  --header "Authorization: Bearer $TOKEN" \
-  --data-raw "{
-    \"orka_vm_name\": \"$VM_NAME\",
+REQUEST_DATA="\"orka_vm_name\": \"$VM_NAME\",
     \"orka_base_image\": \"$BASE_IMAGE\",
     \"orka_image\": \"$VM_NAME\",
     \"orka_cpu_core\": $CPU_COUNT,
     \"vcpu_count\": $VCPU_COUNT,
-    \"vnc_console\": $VNC_CONSOLE
-  }"
+    \"vnc_console\": $VNC_CONSOLE,
+    \"gpu_passthrough\": $GPU_PASSTHROUGH,
```

Comment:
We may want to move the GPU passthrough param to the `orka-deploy` script instead. Since GPU is a much more limited cluster resource (1 per node) it may make sense to specify per VM deploy rather than the VM config, since all VMs deployed in a pipeline with Tekton will use the same VM config.